### PR TITLE
fix(app): disable module commands when protocol paused

### PR DESCRIPTION
**Pull request by sfoster1**
The module command cards displayed in the protocol run page have
historically allowed users to send module commands (like setting
temperature) when the protocol was active but paused. That used to work
essentially as a side effect: modules didn't respect pausing, so you
could tell them to do things while the protocol was paused.

Now that modules respect pausing, sending module commands when the
protocol is paused results in the modules not doing anything.

The science and UX intent of controlling modules from the protocol run
page is to implement pre-protocol preparation (like preheat) or
post-protocol behavior (like keeping some incubation going), not to
control the modules when paused - and in fact, when more advanced module
control behaviors land in protocol designer or in the python api,
controlling modules when the protocol is paused could break protocol
execution.

This PR changes the module command cards to only work when the protocol
is not paused or running.

## Testing
Note: Testing this pr requires #5194 to be installed on a robot (or a robot on 3.16.1 or previous)
- [ ] The module cards in the protocol run page display properly if the proper modules are configured and present
- [ ] Before the protocol starts or after it finishes, the control interface is displayed and functions
- [ ] While the protocol is running and while it is paused, the control interface is not displayed

## Risk Assessment

Limited to out-of-protocol module interaction (and only in the app, at that). However, we should make sure to test both the protocol run page module cards and the pipette info page run cards.

### CODE REVIEW

**Review by nusrat813 — APPROVED** [review]
Working as intended

File: app/src/components/ModuleLiveStatusCards/TempDeckCard.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const TempDeckCard = ({
   >
     <div className={styles.card_row}>
       <StatusItem status={module.status} />
-      {!isProtocolActive && (
+      {allowInteraction && (
```
Comment (mcous) [inline app/src/components/ModuleLiveStatusCards/TempDeckCard.js]: Disappearing this UI suddenly when a protocol is paused goes against #4391 in my mind. Can we use the `disabled` prop of the `<TemperatureControl>` component (and maybe add a `disabledReason` prop to modify the tooltip) instead?

File: app/src/components/ModuleLiveStatusCards/index.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ import { ThermocyclerCard } from './ThermocyclerCard'
 export const ModuleLiveStatusCards = () => {
   const modules = useSelector(getAttachedModulesForConnectedRobot)
   const sendModuleCommand = useSendModuleCommand()
-  const isProtocolActive: boolean = useSelector(robotSelectors.getIsActive)
```
Comment (mcous) [inline app/src/components/ModuleLiveStatusCards/index.js]: This is the only usage of `robotSelectors.getIsActive`. If we don't need it, we might want to remove it

**Review by mcous — COMMENTED** [review]
I'm don't really love disappearing the control UI instead of disabling it with an explanatory tooltip

File: app/src/components/ModuleLiveStatusCards/index.js (hunk 2 above)
Comment (sfoster1) [reply app/src/components/ModuleLiveStatusCards/index.js]: 👍

File: app/src/components/ModuleLiveStatusCards/TempDeckCard.js (hunk 1 above)
Comment (sfoster1) [reply app/src/components/ModuleLiveStatusCards/TempDeckCard.js]: Could we do that in a follow-up pr?

Comment (mcous) [reply app/src/components/ModuleLiveStatusCards/TempDeckCard.js]: This PR is pretty small as is, do you want me to push the change I'm requesting to this branch?

Comment (sfoster1) [reply app/src/components/ModuleLiveStatusCards/TempDeckCard.js]: Yeah please do

**Comment by nusrat813** [comment]
Issues with text on the run screen for module card when the protocol is paused 

Note: The issue is only on run tab

Screen Shot 2020-03-16 at 5 18 29 PM

**Pull request merged**